src/libraries/System.Text.Json/tests/System.Text.Json.SourceGeneration.Tests/RealWorldContextTests.cs

```diff
@@ -399,7 +399,8 @@ protected static ActiveOrUpcomingEvent CreateActiveOrUpcomingEvent()
                 EndDate = DateTime.UtcNow.AddYears(1),
                 Name = "Just a name",
                 ImageUrl = "https://www.dotnetfoundation.org/theme/img/carousel/foundation-diagram-content.png",
-                StartDate = DateTime.UtcNow
+                StartDate = DateTime.UtcNow,
+                Offset = TimeSpan.FromHours(2)
```

Comment:
I believe the method `VerifyActiveOrUpcomingEvent()` should be updated to verify `Offset`.

Reply:
@eiriktsarpalis are you planning on addressing this?